tools/autograph-monitor/monitor.go

```diff
@@ -97,6 +109,21 @@ func main() {
 	}
 }
 
+func rawAutographURLToMonitorEndpoint(autographURLEnvVar string) (string, error) {
+	baseURL, err := url.Parse(autographURLEnvVar)
+	if err != nil {
+		return "", fmt.Errorf("failed to parse AUTOGRAPH_URL as url: %s", err)
+	}
+	if baseURL.Scheme != "https" && baseURL.Scheme != "http" {
+		return "", fmt.Errorf("AUTOGRAPH_URL %#v must be an https:// (or http:// url in testing)", autographURLEnvVar)
```

Comment:
Is there any way to definitively detect a testing environment? If so, we should only allow `http` for that situation.